src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/create-cluster/create-cluster-review.component.html

```diff
@@ -0,0 +1,29 @@
+<div class="row">
+  <div class="col-lg-4">
+    <fieldset>
+      <legend class="cd-header"
+              i18n>Cluster Resources</legend>
+      <table class="table table-striped">
+        <tr>
+          <td i18n
+              class="bold">Hosts</td>
+          <td>{{ hostsCount }}</td>
+        </tr>
+      </table>
+    </fieldset>
+  </div>
+
+  <div class="col-lg-8">
+    <legend i18n
+            class="cd-header">Hosts Per Label</legend>
```

Comment:
`Hosts by Label` or `Hosts per Label` ?

Reply:
To be honest i don't know...I thought maybe `Per` is better. But I can change it.

Reply:
Hosts by Label I guess since we are filtering the Hosts by Label. and b of the by will be a small letter.